public/static/docs/command-reference/list.md

```diff
@@ -0,0 +1,40 @@
+# list
+
+List file structure of a repo including <abbr>data artifacts</abbr>
+
+## Synopsis
+
+```usage
+usage: dvc list [-h] [-q | -v] [-R] [--outputs-only] [--rev [REV]] url [target]
+
+positional arguments:
+  url              Url to the repo. Can be either local fs path or git+ssh/https url
+  target           Path to directory within the repository to list outputs for.
+
+optional arguments:
+  --outputs-only   Show only dvc outputs.
+  --rev [REV]      Git revision (e.g. branch, tag, SHA)
+```
+
+## Description
+
+The command list files and <abbr>data artifacts</abbr> into pointed repo.
```

Comment:
need to expand and elaborate here. No need to repeat just the same phrase we used to above. Even http://man7.org/linux/man-pages/man1/ls.1.html has something meaningful and non-repetitive here.

Would be great to expand on the fact that we show actual data behind DVC-files. That it can be used before `dvc get` to see that file name to path (which is not easy with Github), etc.